### ISSUE TITLE
Scan Gamedata before install?

### ISSUE THREAD

**Issue by pjf** [KSP-CKAN/CKAN#381]
Right now, we scan gamedata _after_ an install. However if a user has changed their GameData contents _before_ installing something, we'll assume any autodetected DLLs are there, when they're really not. Hence we might not install something like, say, ModuleManager, when it's actually absent.

Doing a GameData scan before install is likely just as good, and means we'll catch missing files _immediately_.  We might need to have the scan _not_ save the registry, as this will trigger an inconsistency check, and the user might be installing something to resolve that inconsistency.

TL;DR: Supporting pre-CKAN files makes us cry. We probably just shouldn't.
